perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -151,6 +151,7 @@
 	private static final String LOGIN = "login";
 	private static final String PASSWORD = "password";
 	private static final String STATUS = "status";
+	private static final String GROUP_ASSIGNMENT_ERRORS = "group_assigned_errors";
```

Comment:
I think you should unify constant name and value (`ASSIGNMENT` vs. `assigned`), but I would prefer using `add`, so something like `group_adding_errors`?

Reply:
Done.